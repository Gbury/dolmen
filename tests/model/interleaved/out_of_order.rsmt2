sat
(model
  (define-fun g ((x Int)) Int 1)
  (define-fun f ((x Int)) Int 0)
)
