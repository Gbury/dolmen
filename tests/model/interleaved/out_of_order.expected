File "tests/model/interleaved/out_of_order.smt2", line 3, character 0-20:
3 | (assert (= (f 0) 0))
    ^^^^^^^^^^^^^^^^^^^^
Error The following constant is not defined, and thus
          has no value: `f`
Hint: The interleave mode for checking model requires constants in the model
  file to be defined in the same order as they are declared in the input
  problem file, you might want to try using the `--check-model-mode=full`
  option.
